query-engine/js-connectors/js/pg-js-connector/src/pg.ts

```diff
@@ -54,22 +55,22 @@ class PgQueryable implements Queryable {
     private async performIO(query: Query) {
         const { sql, args: values } = query
 
-        return await this.driver.client.query(sql, values)
+        return await this.client.query(sql, values)
     }
 }
 
-class PgTransaction extends PgQueryable implements Transaction {
-    constructor(connection: pg.PoolClient) {
-        super({ client: connection, isTransaction: true })
+class PgTransaction extends PgQueryable<TransactionClient> implements Transaction {
+    constructor(client: pg.PoolClient) {
+        super(client)
```

Comment:
Isn't that equivalent to leaving the constructor out?

Reply:
Are you referring to the `constructor(client) { super(client) }` in a subclass of a class having a compatible constructor? Yes, it's indeed redundant. I find it easier to read, but feel free to remove it.